client/my-sites/plugins/categories/index.tsx

```diff
@@ -17,26 +16,44 @@ export type Category = {
 	separator?: boolean;
 };
 
+const ALLOWED_CATEGORIES = [
```

Comment:
What's the difference between this `ALLOWED_CATEGORIES` and the one declared in https://github.com/Automattic/wp-calypso/blob/402f24d0d3ec43ddd1d31ea3aaef55945f97bfe3/client/my-sites/plugins/categories/use-categories.tsx#L8 ?

If we need two different lists, can we add a comment right here?

Reply:
Can probably ignore, hoping to remove these soon, see first item in: https://github.com/Automattic/wp-calypso/issues/63030

Reply:
oh, these are new, should have checked the diff before commenting! Ignore me :smile: